db/migrate/20181017200423_add_service_offering_ref_to_portfolio_item.rb

```diff
@@ -0,0 +1,8 @@
+class AddServiceOfferingRefToPortfolioItem < ActiveRecord::Migration[5.1]
+  def change
+    say_with_time "Removing all PortfolioItems, adding a new required field: service_offerings_ref" do
+      PortfolioItem.delete_all
```

Comment:
@brahmanim @hstastna Unfortunately with the switch to using Topology the existing `PortfolioItems` are no longer valid so we will need to delete them.  Sorry about that.  ☹️